ocaml/xe-cli/bash-completion

```diff
@@ -105,7 +105,7 @@ _xe()
                     pvs-cache-storage-*)
                         # Chop off at the third '-' and append 'list'
                         cmd="$(echo ${OLDSTYLE_WORDS[1]} | cut -d- -f1-3)-list";;
-                    host-cpu-*|host-crashdump-*|gpu-group-*|vgpu-type-*|pvs-server-*|pvs-proxy-*|pvs-site-*|sdn-controller-*)
+                    host-cpu-*|host-crashdump-*|gpu-group-*|vgpu-type-*|pvs-server-*|pvs-proxy-*|pvs-site-*|sdn-controller-*|network-sriov-*)
```

Comment:
I would suggest to split this up to avoid an ever longer line. Two possible solutions:
```
host-cpu-*|host-crashdump-*|gpu-group-*|vgpu-type-*)
    cmd="$(echo ${OLDSTYLE_WORDS[1]} | cut -d- -f1-2)-list";;
pvs-server-*|pvs-proxy-*|pvs-site-*|sdn-controller-*|network-sriov-*)
    cmd="$(echo ${OLDSTYLE_WORDS[1]} | cut -d- -f1-2)-list";;
```

or

```
host-cpu-*|\
host-crashdump-*|\
...
network-sriov-*)
    cmd="$(echo ${OLDSTYLE_WORDS[1]} | cut -d- -f1-2)-list";;
```

Reply:
Thanks for your review @lindig 
Yes, actually it's better to split this up to a long line in future. 
I prefer the second one, will update later. :)

Reply:
Updated, could you please help to look again? @lindig 
Thanks.